test/testQuitDriverAfterEachIteration.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
Yes, it does fail without my change. Please see line #12 where I've configured Click events as one of the arguments. The code does reach line #198 with this configuration and fails (without my change) when driver.getSession() returns undefined value, in cases where we have multiple test cases in the suite.